src/App.vue

```diff
@@ -22,6 +22,12 @@
       @retry="fetchData()"
     />
 
+    <invalid-dissolution-dialog
```

Comment:
I recommend changing our component elements to CamelCase (this one and the others, at least in this file).

Reply:
Will do 👍

Reply:
Updated in next commit 👍